datastore/mongodb/filter_outputs.go

```diff
@@ -29,6 +42,7 @@ func (c *Client) CreateFilterOutput(ctx context.Context, f *model.FilterOutput)
 
 // UpdateFilterOutput creates/updates a FilterOutputs in the CantabularFilters collection
 func (c *Client) UpdateFilterOutput(ctx context.Context, f *model.FilterOutput) error {
+
```

Comment:
Again, tempting as  it may be try to avoid adding blank lines at the beginning of functions.. Once you stick to one method for long enough it will start to always look right

Reply:
removed